delfin/drivers/fujitsu/eternus/eternus_stor.py

```diff
@@ -352,3 +391,527 @@ def parse_alert(context, alert):
+        host_fc_list = self.get_data(command)
+        for host_fc in host_fc_list:
+            if len(host_fc) < encapsulate_data_total:
+                continue
+            fc_name = host_fc[name_count]
+            os = host_fc[type_count].lower()
+            state = host_status.get(fc_name)
+            host_d = {
+                "name": fc_name,
+                "storage_id": self.storage_id,
+                "native_storage_host_id": fc_name,
+                "os_type": consts.HOST_OS_TYPES_MAP.get(
+                    os, constants.HostOSTypes.UNKNOWN),
+                "status": constants.HostStatus.NORMAL if
+                state is not None and state ==
+                consts.HOST_PATH_STATUS_SPECIFIC_TWO else
+                constants.HostStatus.OFFLINE
+            }
+            host_list.append(host_d)
+
+    def get_iscsi_host(self, host_list, host_status):
+        host_iscsi_list = self.get_iscsi_host_data()
+        for host_iscsi in host_iscsi_list:
+            iscsi_name = host_iscsi.get('name')
+            state = host_status.get(iscsi_name)
+            os = host_iscsi.get('os')
+            os = os.lower() if os else None
+            host_d = {
+                "name": iscsi_name,
+                "storage_id": self.storage_id,
+                "native_storage_host_id": iscsi_name,
+                "os_type": consts.HOST_OS_TYPES_MAP.get(
+                    os, constants.HostOSTypes.UNKNOWN),
+                "status": constants.HostStatus.NORMAL if
+                state is not None and state ==
+                consts.HOST_PATH_STATUS_SPECIFIC_TWO else
+                constants.HostStatus.OFFLINE,
+                'ip_address': host_iscsi.get('address')
+            }
+            host_list.append(host_d)
+
+    def get_data(self, command):
+        host_list = []
+        host_str = self.cli_handler.exec_command(command)
+        block = True
+        length_list = []
+        if host_str:
+            host_arr = host_str.strip().replace('\r', '').split('\n')
+            for host_row_str in host_arr:
+                if not host_row_str or \
+                        consts.CLI_STR in host_row_str:
+                    continue
+                if consts.SPECIAL_CHARACTERS_TWO in host_row_str:
+                    identify_list = host_row_str.split()
+                    for identify in identify_list:
+                        length_list.append(len(identify))
+                    block = False
+                    continue
+                if block:
+                    continue
+                volume_list = []
+                key_length = DIGITAL_CONSTANT.ZERO_INT
+                for length_key in length_list:
+                    volume = host_row_str[key_length:
+                                          key_length + length_key].strip()
+                    volume_list.append(volume)
+                    key_length = key_length + length_key + DIGITAL_CONSTANT \
```

Comment:
It is recommended that the code be placed in the next line for better readability.
![image](https://user-images.githubusercontent.com/59107077/153858253-b3ae1cf4-60ed-41a2-8119-2910e4dc6f01.png)

Reply:
Has been modified